robottelo/ui/locators/common.py

```diff
@@ -75,7 +75,9 @@
 
     "search": (By.ID, "search"),
     "auto_search": (
-        By.XPATH, "//ul[contains(@id, 'ui-id')]/li/a[contains(., '%s')]"),
+        By.XPATH,
```

Comment:
That will also require running auto-complete tests for organization and location